tests/console/yast2_ftp.pm

```diff
@@ -0,0 +1,111 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "consoletest";
+use testapi;
+
+
+
+sub run() {
+
+    select_console 'root-console';
+
+    # install vsftps
+    assert_script_run("/usr/bin/zypper -n -q in vsftpd");
+
+    # create DSA certificate for ftp server at first which canbe used for SSL configuration
+    type_string("openssl req -x509 -nodes -days 365 -newkey rsa:1024 -keyout /etc/vsftpd.pem -out /etc/vsftpd.pem\n");
+    sleep 2;
+    type_string "DE\n";
+    type_string "bayern\n";
+    type_string "nuremberg\n";
+    type_string "SUSE\n";
+    type_string "QA\n";
+    type_string "localhost\n";
+    type_string "admin\@localhost\n";
+
+
+    assert_script_run("ls -l /etc/vsftpd.pem");    # check vsftpd.pem is created
+
+    # start yast2 apache2 configuration
+    script_run("/sbin/yast2 ftp-server; echo yast2-ftp-server-status-\$? > /dev/$serialdev", 0);
+    assert_screen 'ftp-server';                    # check ftp server configuration page
```

Comment:
I don't think you should redo this one here, but for future work: please consider that 'ftp-server' is a very generic term. While your needle here is looking for yast2 - but none of your tags contain the word yast2. Better have longer tag names and avoid namespace collisions.